Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/PredictorCorrector.java

```diff
@@ -417,7 +419,11 @@ protected <P extends AnomalyDescriptor> boolean isSignificant(boolean significan
                 }
                 answer = significantScore || (delta > shiftAmount + DEFAULT_NORMALIZATION_PRECISION);
                 if (answer) {
-                    answer = (a < b - ignoreNearExpectedFromBelow[y]) || (a > b + ignoreNearExpectedFromAbove[y]);
+                    boolean lower = (a < b - ignoreNearExpectedFromBelow[y])
+                            && (a < b * (1 - ignoreNearExpectedFromBelowByRatio[y]));
+                    boolean upper = (a > b + ignoreNearExpectedFromAbove[y])
+                            && (a > b * (1 + ignoreNearExpectedFromBelowByRatio[y]));
```

Comment:
Should we use ignoreNearExpectedFromAboveByRatio[y]?

Reply:
Yes ;) It's fixed in https://github.com/aws/random-cut-forest-by-aws/pull/389/commits/424ae83b67ef5f6620e48534e918799f70eb80fb